From 9d2a896b1b9817c53e443a80aa689cbe49d64c23 Mon Sep 17 00:00:00 2001
From: Julien Gros <julien.gros@intel.com>
Date: Mon, 21 Jan 2013 15:59:52 +0100
Subject: [PORT FROM MAIN][BT] Change Bluetooth permissions for UIM

BZ: 81675

Needed to avoid running bd_prov as root.

Change-Id: I7f501aadd22ca434def15121b7a065de02afd4b5
Orig-Change-Id: I6550357e1a78d005befe70b89d20e8672ac22304
Feature: BT
Category: Improvement
Purpose: AOSP+
Origin: Internal
Upstream-candidate: no
Signed-off-by: Julien Gros <julien.gros@intel.com>
Reviewed-on: http://android.intel.com:8080/87782
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 init/property_service.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index 384a414..9170da0 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -108,6 +108,7 @@ struct {
     { "dumpstate",AID_SHELL, AID_LOG },
     { "ril-daemon",AID_RADIO, AID_RADIO },
     { "pcsc",AID_WIFI, AID_WIFI },  /* Allow wpa_supplicant to start the pcsc-lite daemon used for EAP-SIM/AKA auth */
+    { "uim",AID_BLUETOOTH, AID_BLUETOOTH },
      {NULL, 0, 0 }
 };
 
-- 
1.7.4.1

